### PR TITLE
Feature/add ask question btn

### PR DESCRIPTION
**Pull request by joslinchan**
**Work done:**
- added "Ask a question' button using semantic-ui-react components
- converted AskQuestionForm to a model using components from above library

**Screenshot**
Fs3ZGWAgoI

### CODE REVIEW

File: client/src/components/NavLink.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ class Menu extends Component {
     return (
       
         <div>
-        <Button id="menu" class="ui icon button" onClick={this.showMenu}> <i class="bars icon" onClick={this.showMenu}></i>  
+        <Button id="menu" className="ui icon button" onClick={this.showMenu}> <i className="bars icon" onClick={this.showMenu}></i>
```
Comment (aviskarkc10) [inline client/src/components/NavLink.js]: thanks for making this better :+1:

File: client/src/containers/AddQuestionForm.js [hunk 2]
```diff
@@ -1,50 +1,89 @@
...
+import React, { Component, createRef } from 'react';
+import { Button, Form, Icon, Modal, ModalActions } from 'semantic-ui-react';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+import { postQuestion } from '../actions';
+import '../styles/AskQuestionForm.css';
 
 class AddQuestionForm extends Component {
-  state={}
+  state = {};
```
Comment (aviskarkc10) [inline client/src/containers/AddQuestionForm.js]: since we are using `constructor` to assign state, we can remove this

**Comment by nthopinion** [comment]
@joslinchan There is some conflict. Did you remove 
import { Grid, Ref } from "semantic-UI-react";
Also how is it looking in a responsive mobile view?

**Comment by nthopinion** [comment]
Thanks Mergig now...

**Comment by joslinchan** [comment]
The modal is looking pretty good in mobile/responsive state. 
I currently have the button in the sticky header but that can be moved.

I'm not sure if this is needed, but we might want to look into adding a theme file to customize the colors. Right now I'm using the blue from semantics-ui-react since it's pretty close to the blue we want- but overwriting style does not seem sustainable as this project gets bigger.

**Comment by nthopinion** [comment]
> 
> 
> The modal is looking pretty good in mobile/responsive state.
> I currently have the button in the sticky header but that can be moved.
> 
> I'm not sure if this is needed, but we might want to look into adding a theme file to customize the colors. Right now I'm using the blue from semantics-ui-react since it's pretty close to the blue we want- but overwriting style does not seem sustainable as this project gets bigger.

Let me deploy it to a different site and we can do a quick review

**Comment by nthopinion** [comment]
https://ntozwu-pd-covid19-app.azurewebsites.net/ check this out

**Pull request merged**